packages/react-components/src/molecules/SocialIcons.tsx

```diff
@@ -12,29 +12,43 @@ import {
   researcherIdIcon,
   orcidSocialIcon,
   googleScholarIcon,
+  globeIcon,
 } from '../icons';
 import { perRem } from '../pixels';
 import { tin } from '../colors';
 
+const ROW_GAP = 12;
 const socialContainerStyles = css({
-  display: 'grid',
-  columnGap: `${12 / perRem}em`,
-  gridAutoFlow: 'column',
+  display: 'flex',
+  flexWrap: 'wrap',
+  paddingRight: `-${ROW_GAP / perRem}em`,
 });
 
 const iconStyles = css({
   paddingTop: `${15 / perRem}em`,
   paddingBottom: `${17 / perRem}em`,
+  paddingRight: `${ROW_GAP / perRem}em`,
 });
 
+const fillInactiveStyles = css({ svg: { fill: tin.rgb } });
+const strokeInactiveStyles = css({ svg: { stroke: tin.rgb } });
+
 const SocialIconLink: React.FC<{
   link?: string;
   icon: JSX.Element;
+  strokeInactive?: boolean;
   isOwnProfile: boolean;
-}> = ({ link, icon, isOwnProfile }) =>
+}> = ({ link, icon, isOwnProfile, strokeInactive = false }) =>
```

Comment:
Urgh yeah we have another problem here, I remember seeing this briefly. The user of this should not have to decide how to transform the icon, each icon should know that on its own. This will also require some though probably